itests/setup.py

```diff
@@ -60,21 +56,24 @@ def _wait_until_accept(port, timeout=3.0):
 @contextmanager
 def api_server(tmpdir):
     # type: (LocalPath) -> Iterator[str]
-    api_port = _get_unused_port()
+    api_socket = _bind_socket()
+    api_port = api_socket.getsockname()[1]
 
     cmd = [
         sys.executable,
         src_path("bin", "grouper-api"),
+        "-v",
+        "-v",
```

Comment:
Nit: "-vv"?

Reply:
Yeah, I'll make that consistent with the other ones that use `-vvc`. (Two `-v` is required to make it log at debug level, IIRC.)